Auth0Tests/CredentialsSpec.swift

```diff
@@ -50,27 +49,17 @@ class CredentialsSpec: QuickSpec {
                 expect(credentials.recoveryCode) == RecoveryCode
             }
 
-            it("should have only access_token and token_type") {
-                let credentials = Credentials(json: ["access_token": AccessToken, "token_type": Bearer])
-                expect(credentials).toNot(beNil())
+            it("should have only access_token, id_token, and token_type") {
+                let credentials = Credentials(json: ["access_token": AccessToken, "id_token": IdToken, "token_type": Bearer])
                 expect(credentials.accessToken) == AccessToken
                 expect(credentials.tokenType) == Bearer
-                expect(credentials.idToken).to(beNil())
-                expect(credentials.expiresIn).to(beNil())
+                expect(credentials.idToken) == IdToken
+                expect(credentials.refreshToken).to(beNil())
+                expect(credentials.expiresIn).to(beCloseTo(Date(), within: 5))
                 expect(credentials.scope).to(beNil())
                 expect(credentials.recoveryCode).to(beNil())
             }
 
-            it("should have id_token") {
```

Comment:
All properties (nullable and non-nullable) are tested in the two previous test cases.